gitpod-io/workspace-images
# Set max.workers for Gradle to fit for the selected workspace class

## Description
Gradle sets the max number of workers to the number of processors. Unfortunately, it take the number of processors available on the node (32) and not the number of processors available in the workspace.

## Related Issue(s)
Fixes # ENT-226

## How to test
Steps to reproduce
1. Open a Gitpod workspace on a repo that has a Gradle build.
2. run ./gradlew -i
Observe line `Using 32 worker leases` among its output.

Workaround `run /gradle -i --max-workers 4`
1. Observe line `Using 4 worker leases` among its output.

## Documentation


/hold
